patches/patch1.diff

```diff
@@ -0,0 +1,19230 @@
+diff --git a/help/C/basics.page b/help/C/basics.page
```

Comment:
Could you please add a comment section at the top of this patch file saying why it's needed/what it's for?

Reply:
Done.